app/package.json

```diff
@@ -118,6 +118,7 @@
     "nyc": "^14.1.1",
     "prerender-spa-plugin": "^3.4.0",
     "puppeteer": "^1.18.1",
+    "sass": "^1.17.4",
```

Comment:
Why is this necessary? Was it mentioned in the vuetify docs.?
If so, pls update it to the latest version

Reply:
> Why is this necessary? Was it mentioned in the vuetify docs.?
> If so, pls update it to the latest version

Yes this is mentioned on the docs. I updated the version to 1.22.3